en/docs/references/connect-with-protected-third-party-applications.md

```diff
@@ -2,7 +2,7 @@
 
 When your Choreo component connects with protected third-party applications, the firewalls of these applications might block the requests of the Choreo component. To address this, you can whitelist the following public IP ranges of Choreo that it assigns to Choreo components as client IPs:
```

Comment:
When your Choreo components connect with protected third party applications, the firewalls of these applications might block the requests of the Choreo component. To address this, you can whitelist the following public IP ranges of Choreo that it assigns to Choreo components as client IPs: